src/schema/schema.rs

```diff
@@ -104,10 +105,199 @@ impl Schema {
           Some(error) => error
         }
       },
-      (&SchemaType::Enum(_), &Query::Keys(_)) => Err(Error::validation("Cannot deeply query an enum.", NO_PRIMITIVE_HINT)),
+      (&SchemaType::Enum{..}, &Query::Keys(_)) => Err(Error::validation("Cannot deeply query an enum.", NO_PRIMITIVE_HINT)),
+      (&SchemaType::None, _) |
       (_, &Query::All) => Ok(())
     }
   }
+
+  /// Create a schema which does not run any validations.
+  pub fn none() -> Self {
```

Comment:
Also, users may depend on these functions. I was thinking of names like `new_*` or `create_*` but chose not to go that way because the single word is cleaner, and makes sense when combined with the type definition.

If this isn't the *best* way to do it, that's fine. We can add new ways to create schemas in the future. We just can't ever remove this code.

As a side note, I checked, clippy has no opinions on the format for these methods 😖